scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
That is a good question. For curiosity, I just tried with ct-ng on a Linux (Fedora 25) host. It used plain gcc for the GMP build as well. It looks like GMP's configure is behaving badly in this respect.